frameworks/projects/Core/src/main/royale/org/apache/royale/core/LayoutBase.as

```diff
@@ -310,7 +310,6 @@ package org.apache.royale.core
 		 *  @playerversion Flash 10.2
 		 *  @playerversion AIR 2.6
 		 *  @productversion Royale 0.8
-		 * @royaleignorecoercion org.apache.royale.core.ILayoutParent
```

Comment:
Why was it necessary to remove this?

Reply:
It wasn't strictly necessary.  I could put it back in, if that's better.  I took it out because there were hosts not implementing ILayoutParent (but were implementing getLayoutHost()), and this wasn't being caught anywhere on the JS side (only SWF side).  I thought it would be helpful if somewhere in LayoutBase it caught that, and this seemed the best place.  Since casting is a much cheaper operation now, I didn't think it would be bad for performance, but I haven't measured that.